### PR TITLE
Use memcached for slow functions

### PR DESCRIPTION
**Pull request by jbeezley**
This reduces the response time for pca plots on the client by speeding up `GET /csv/:id`, `GET /csv/:id/plot/pca`, and `PUT /csv/:id/normalizaton` by ~10x--making the update when toggling normalization take about 100ms.

### CODE REVIEW

File: metabulo/views.py [hunk 1]
```diff
@@ -118,7 +125,7 @@ def set_normalization_method(csv_id):
         csv_file.normalization = method
         db.session.add(csv_file)
         db.session.commit()
-        return jsonify(csv_file_schema.dump(csv_file))
+        return jsonify(method)
```
Comment (subdavis) [inline metabulo/views.py]: @jbeezley @waxlamp  could we revert this change?

I'm not aware of any scenario from the UI where this speeds things up.  Any time the norm/trans/scaling are changed, the UI will need to reload the CSV to refresh the graphs.  There are no scenarios where this endpoint will be hit without the other.

Breaking this into 2 API requests just adds extra latency.

Comment (waxlamp) [reply metabulo/views.py]: Some context: I need to use the transformed values for the loadings plot, and was getting the name of the last applied transform/normalization/scaling operator in the `dataset.transformed` field, rather than the transformed data itself. As things stand now (after this change was made), `dataset.transformed` carries just about zero information.

Comment (jbeezley) [reply metabulo/views.py]: It's fine with me, I'm too far removed to remember why I did it in the first place.

Comment (waxlamp) [reply metabulo/views.py]: Cool. Rather than literally revert this change, I'll do it as part of my current branch.

Thanks @subdavis and @jbeezley!
